test/integration/delta_cds_integration_test.cc

```diff
@@ -0,0 +1,213 @@
+#include "envoy/api/v2/cds.pb.h"
+#include "envoy/api/v2/discovery.pb.h"
+#include "envoy/grpc/status.h"
+#include "envoy/stats/scope.h"
+
+#include "common/config/protobuf_link_hacks.h"
+#include "common/config/resources.h"
+#include "common/protobuf/protobuf.h"
+#include "common/protobuf/utility.h"
+
+#include "test/common/grpc/grpc_client_integration.h"
+#include "test/integration/http_integration.h"
+#include "test/integration/utility.h"
+#include "test/mocks/server/mocks.h"
+#include "test/test_common/network_utility.h"
+#include "test/test_common/simulated_time_system.h"
+#include "test/test_common/utility.h"
+
+#include "absl/synchronization/notification.h"
+#include "gtest/gtest.h"
+
+using testing::AssertionFailure;
+using testing::AssertionResult;
+using testing::AssertionSuccess;
+using testing::IsSubstring;
+
+namespace Envoy {
+namespace {
+
+// TODO(fredlas) Move to test/config/utility.cc once there are other xDS tests that use gRPC.
```

Comment:
Is now the time to make the move, since we have this in `cds_integration_test`?

Reply:
Sure, but in a different PR; this one is already plenty big.

Reply:
OK, but I would really prefer we do this soon, it's hard to maintain these boiler plate fragments across tests.

Reply:
Done, in fact, was able to have them share significantly more than the TODO'd parts. See https://github.com/fredlas/envoy/commit/00d9b7c21e2c7b2ee21d62f6fae78c524c726adc